### ISSUE TITLE
Pretty Page only displaying code *below* the error

### ISSUE THREAD

**Issue by j-applese3d** [filp/whoops#699]
Like @davidrans [commented](https://github.com/filp/whoops/commit/fdf92f03e150ed84d5967a833ae93abffac0315b#commitcomment-53752503), in v2.14.0 the snippet of displayed code is not showing the lines surrounding the error, but just what comes directly after it.

Ex:
```
<?php

require_once 'vendor/autoload.php';
if (! \Whoops\Util\Misc::isAjaxRequest())
{
    $whoops = new \Whoops\Run;
    $whoops->pushHandler(new \Whoops\Handler\PrettyPageHandler);
    $whoops->register();
}

throw new Exception("Test");
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
// Here are some comments after the exception
```

Shows:
![image](https://user-images.githubusercontent.com/24463871/127373400-ab625506-5500-4572-a38a-0e3999be81c0.png)

I believe this is the problematic change: https://github.com/filp/whoops/commit/7376a422a13e8cd738dacb2518976f25c4ffc3c0#r54103819

**Comment by SergioMendolia** [comment]
I came here to report the same issue

**Comment by qodo** [comment]
I am seeing this issue too! It no longer shows the error but the code surrounding it.

**Comment by denis-sokolov** [comment]
Your thumbs up emoji on the first comment of the issue are appreciated. Additional comments without any new information – less beneficial!

**Comment by qodo** [comment]
Sorry! I didn't have much more to offer beyond @j-applese3d comment. Something that might help: removing the `max-height: 345px;` makes the line in question viewable.

**Comment by sebkpf** [comment]
I am still on v2.3 and having the same issue since about a week.

**Comment by j-applese3d** [comment]
Adjusting the `highlightCurrentLine` function to be:

```
  var highlightCurrentLine = function() {
    // We show more code than needed, purely for proper syntax highlighting
    // Let’s hide a big chunk of that code and then scroll the remaining block
    $activeFrame.find('.code-block').first().css({
      maxHeight: 345,
      overflow: 'hidden',
    });  // removed .scrollTop(999)

    // added the following 3 lines:
    highlightedLine = $activeFrame.find('.code-block .line-highlight').first()[0];
    highlightedLine.scrollIntoView();
    highlightedLine.parentElement.scrollTop -= 180;

    $container.scrollTop(0);
  }
```

Appears to resolve the problem.  It scrolls the div containing the code to the top of the highlighted line, and then goes back up 180px so that the highlighted line is near the middle.

**Comment by futureisrise** [comment]
Same problem

**Comment by denis-sokolov** [comment]
Fix released in 2.14.1.

@j-applese3d, thank you, your fix added in 13f8171.
